### DISCUSSION TITLE
Is there a way to convert a string into a list of KeyCode?

### DISCUSSION THREAD

**Discussion by igngi** [TolikPylypchuk/SharpHook#53] [Q&A] [answered]
I need to simulate a user typing an arbitrary string. Is it possible to go from string to KeyCodes so I can use SimulateKeyPress/Release?

Thanks a lot!

**Comment by TolikPylypchuk** [comment, marked as answer]
Hi! Thanks for asking! I'm currently investigating this feature, and it appears that it's possible to do on all three OSes. There's a good chance that this will be added in version 5.0.0 which I will release soon. No promises though.

**Reply by igngi** [reply]
Oh, very cool. Thanks!

**Reply by TolikPylypchuk** [reply]
FYI, I've implemented text simulation, and it will be available in version 5.0.0 which I'll release soon.

**Reply by igngi** [reply]
Oh, that’s fantastic, thank you. Does it support any unicode string?

**Reply by TolikPylypchuk** [reply]
Yes, it supports the entirety of UTF-16 (even emojis and surrogate pairs in general).

**Reply by TolikPylypchuk** [reply]
@igngi I've just released version 5.0.0 which adds support for text entry simulation.
